aggregates/governance-aggr/src/main/java/com/bloxbean/cardano/yaci/store/governanceaggr/storage/impl/DRepStorageReaderImpl.java

```diff
@@ -238,7 +238,7 @@ public Optional<DRepDetailsDto> getDRepDetailsByDRepId(String drepId, int epoch)
                 ? DRepStatus.ACTIVE
                 : DRepStatus.INACTIVE;
 
-        if (status == DRepStatus.INACTIVE || status == DRepStatus.RETIRED) {
```

Comment:
Does this mean that an inactive dRep will also have voting power?

Reply:
@Sotatek-HuyLe3a I think we can keep voting power as is. Amount may create more confusion.